src/applications/ask-va/containers/DashboardCards.jsx

```diff
@@ -1,23 +1,35 @@
 import {
   VaAlert,
+  VaCard,
+  VaLink,
+  VaLoadingIndicator,
   VaSelect,
 } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
 import { apiRequest } from '@department-of-veterans-affairs/platform-utilities/api';
+import { format, parse } from 'date-fns';
 import moment from 'moment';
 import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router';
+import { Tab, TabList, TabPanel, Tabs } from 'react-tabs';
 import { ServerErrorAlert } from '../config/helpers';
 import { envUrl } from '../constants';
+import { mockInquiryData } from '../utils/mockData';
 
 const DashboardCards = () => {
   const [error, hasError] = useState(false);
   const [inquiries, setInquiries] = useState([]);
+  const [, setOriginalInquries] = useState([]);
```

Comment:
State variable?

Reply:
It's due to the state, originalInquiries, not being used anywhere at the moment since we are mocking the data for the api call. It will get added once we're using live data.